From dbd56e6ce52549c91862853de39c7d33055fcec6 Mon Sep 17 00:00:00 2001
From: "Valery V. Vorotyntsev" <valery_vorotyntsev@xyratex.com>
Date: Thu, 22 Mar 2012 13:37:12 +0200
Subject: [PATCH 086/370] remove `NUMA Optimizations' subsection

This [mandatory] section would add nothing but formal noise.

vvv:    Have no idea what to say here. Your suggestions are welcome.

carl:   In user space, the confc state machine activities are operated
        under the control of the request handler, and hence inherits [sic]
        the latter's NUMA optimizations.

        (at least I think this is true)

        In the kernel, I don't know!  I'm clueless on the request handler
        replacement.  Perhaps Nikita can chime in?

nikita: [* blink *] Why do we need this section at all?

Reviewed-on: http://reviewboard.clusterstor.com/r/714/
---
 conf/confc.c | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/conf/confc.c b/conf/confc.c
index 613689a..e8d603c 100644
--- a/conf/confc.c
+++ b/conf/confc.c
@@ -41,7 +41,6 @@
  *   - @ref confc-dld-lspec-comps
  *   - @ref confc-dld-lspec-state
  *   - @ref confc-dld-lspec-thread
- *   - @ref confc-dld-lspec-numa
  * - @ref confc-dld-conformance
  * - @ref confc-dld-ut
  *   - @ref confc-dld-ut-infra
@@ -254,10 +253,6 @@
  * - updating c2_conf_obj::co_status;
  * - updating c2_conf_obj::co_nrefs.
  *
- * @subsection confc-dld-lspec-numa NUMA Optimizations
- *
- * XXX Have no idea what to say here. Your suggestions are welcome.
- *
  * <hr> <!------------------------------------------------------------->
  * @section confc-dld-conformance Conformance
  *
-- 
1.8.3.2

